candidate-standard/requirements/edr/query_type/item.adoc

```diff
@@ -1,30 +1,47 @@
-===== Parameters
+The EDR Items query is an <<ogc17-069r3,OGC API - Features>> endpoint that 
+may be used to catalog pre-existing EDR sampling features. 
+The pre-existence of an EDR sampling feature may be because of the existence of
+a monitoring location, because a particular query has been cached for later use, or
+for an array of application-specific use cases (e.g. a catalog of spatio-temporal domains of anomalies in a dataset). A https://github.com/opengeospatial/Environmental-Data-Retrieval-API/blob/master/candidate-standard/openapi/EDR_OpenAPI.yaml#L2424[GeoJSON-compatible 
+JSON-Schema] has been specified to document an EDR query endpoint and valid 
+query parameters including time range, parameters, and spatial characteristics.  
 
-If an itemID is not specified the query will return a list of the available
-itemID's with a description of their content in a GeoJSON format.
+include::../../../recommendations/core/REC_edr-geojson.adoc[]
 
+===== Parameter itemID
```

Comment:
I think we want to do the path parameters like this? Do we need a requirement for path parameters too?

Reply:
I not sure that we need to constrain the definition of the `{itemId}` beyond the fact that it needs to be a string which is a valid component of a URI

Reply:
@dblodgett-usgs I agree with @m-burgoyne : an Id should just be a URI valid string.

Reply:
Yes. See: https://github.com/opengeospatial/Environmental-Data-Retrieval-API/issues/108